atlasdb-api/src/main/java/com/palantir/atlasdb/keyvalue/api/ColumnSelection.java

```diff
@@ -44,12 +43,12 @@ private ColumnSelection(SortedSet<byte[]> selectedColumns) {
 
     public static ColumnSelection valueOf(String serialized) {
         Set<byte[]> columns = Sets.newTreeSet(UnsignedBytes.lexicographicalComparator());
-        for (String strColumn : serialized.split("\\s*,\\s*")) {
-            strColumn = strColumn.trim();
-            if (strColumn.equals("")) {
+        for (String columnString : serialized.split("\\s*,\\s*")) {
+            String trimmedColumnString = columnString.trim();
```

Comment:
Yes, I agree. I'm trying to keep the PR limited to just making everything baseline compliant except small things I see along the way (such as where it looks like something could fail but when you dig a few layers deep you find out that it won't because of a Preconditions, etc)